ued/util.py

```diff
@@ -39,5 +40,5 @@ def get_signal_by_pvname(pvname: str) -> ophyd.EpicsSignal:
             return sig
 
     if pvname not in _pv_cache:
-        _pv_cache[pvname] = Motor(pvname, name=pvname)
```

Comment:
🤦‍♂️ @klauer

Reply:
At least now you know that this was never used

Reply:
They probably stuck to the same few known motors